sql/2019/06_Fonts/06_03.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+
+-- counts the font types (format)
+
+WITH
+  summary_requests AS (
+    SELECT * FROM `httparchive.summary_requests.2019_07_01_desktop`
+    UNION ALL
+    SELECT * FROM `httparchive.summary_requests.2019_07_01_mobile`
+  ),
+  processedTable AS (
+    SELECT
+      (
+        IF(STRPOS(mimeType, "font") > 0,
+          -- get the type of font from the content-type
```

Comment:
Ahh! We actually have an `ext` field for the file extension which is probably a good enough heuristic for the font format.

```sql
#standardSQL
SELECT
  DISTINCT LOWER(ext),
  COUNT(0) OVER (PARTITION BY LOWER(ext)) / COUNT(0) OVER () AS pct
FROM
  `httparchive.summary_requests.2019_07_01_*`
WHERE
  type = 'font'
ORDER BY
  pct DESC
```

Your logic would be good to include in the `format` field function: https://github.com/HTTPArchive/legacy.httparchive.org/blob/31a25b9064a365d746d4811a1d6dda516c0e4985/bulktest/batch_lib.inc#L994

Reply:
But would it not get most of them? (given not all are set as type font?)

Reply:
The results of your query vs mine are very similar so in that case I'd prefer the simpler solution.